src/data/localStorage.ts

```diff
@@ -111,21 +111,23 @@ export const restoreFromLocalStorage = () => {
   if (savedElements) {
     try {
       elements = JSON.parse(savedElements);
-    } catch {
+    } catch (error) {
+      console.error(error);
```

Comment:
do we want to send it to sentry coz this might unnecessary pollute sentry as it will be json parse error

Reply:
Not sure. While the error will be mostly useless, we'll at least know something is wrong.

Reply:
I think it will be always useless due to JSON parse, we can keep it now and may remove it later if it starts polluting sentry